_targets.R

```diff
@@ -101,5 +101,27 @@ list(
                   file_out = "flow_cartogram.png",
                   width = 16),
     format = "file"
+  ),
+  
+  # Flow timeseries nationally - instagram 
+  tar_target(
+    flow_national_instagram_png,
+    national_ig(file_svg = "flow_national_ig.png", 
+                  plot_left = plot_nat, 
+                  date_start,
+                  width = 1080, height = 1080, color_bknd),
```

Comment:
Since the national chart and state tiles are on separate outputs, it would be smart to rename those targets to be more informative & broadly applicable to export functions